docs/sources/outgoing-webhooks/_index.md

```diff
@@ -356,23 +355,23 @@ Take this template for example:
 
 ```json
 {
-  "labels" : "{{ alert_payload.labels }}"
+  "labels": "{{ alert_payload.labels }}"
 }
 ```
 
 It will result in the following (Invalid JSON due to single quotes):
 
 ```json
 {
-  "labels": {'region': 'eu-1', 'alertname': 'TestAlert'}
```

Comment:
This was intentionally wrong to demonstrate the result if you don't use `tojson()`